mac

```diff
@@ -155,11 +154,15 @@ if brew list | grep --silent "qt@5.5"; then
   brew link --force qt@5.5
 fi
 
+if [ "$(which psql)" == "" ] ; then
+  fancy_echo "'psql' not found in PATH, attempting to add it..."
+  append_to_bash_profile 'export PATH="/usr/local/opt/postgresql@9.6/bin:$PATH"'
+fi
+
 fancy_echo "Configuring asdf version manager..."
 if [ ! -d "$HOME/.asdf" ]; then
   git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.4.2
-  append_to_bash_profile "source $HOME/.asdf/asdf.sh" 1
-  append_to_bash_profile "source $HOME/.asdf/completions/asdf.bash" 1
+  append_to_bash_profile 'export PATH="$HOME/.asdf/shims:$HOME/.asdf/bin:$PATH"' 1
```

Comment:
It's better to prepend, rather than `source` because of how `.bash_profile` is run when opening a VS Code terminal.

For some reason, it's run twice, so everything that prepends does so twice. This puts a lot of things in front of `asdf` (since `source`ing twice does nothing), which causes issues.